manifests/03-rbac-role-ns-openshift-config-managed.yaml

```diff
@@ -12,3 +12,21 @@ rules:
   - get
   - list
   - watch
+  - create
+  - update
+  - delete
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: console-config-public
```

Comment:
@enj I followed the naming for our existing console operator roles here, but it looks like we should be using a `system:` prefix?

Reply:
I think this is fine.  It does not escape the namespace and nothing else needs to bind to this role.